### PR TITLE
Missing methods

### PR DESCRIPTION
**Pull request by hadley**
Part of #661

### CODE REVIEW

**Comment by hadley** [comment]
Better to fix upstream

**Comment by hadley** [comment]
Worth doing a quick fix before we fully patch.

File: R/db.R [hunk 1]
```diff
@@ -365,6 +376,20 @@ setMethod(
...
+#' @rdname SQLServer
+#' @usage NULL
+setMethod(
+  "dbExistsTable", c("Microsoft SQL Server", "SQL"),
+  function(conn, name, ...) {
+    dbExistsTable(
+      conn,
+      name = id_field(name, "table"),
+      catalog_name = id_field(name, "catalog"),
+      schema_name = id_field(name, "schema")
```
Comment (hadley) [inline R/db.R:388]: This is ID not SQL

**Review by detule — APPROVED** [review]
LGTM for a near term solution.  I like where your ideas for a longer term solution.

**Pull request merged**